cli-plugins/socket/socket_nodarwin.go

```diff
@@ -13,7 +13,7 @@ func listen(socketname string) (*net.UnixListener, error) {
 	})
 }
 
-func onAccept(conn *net.UnixConn, listener *net.UnixListener) {
+func unlink(listener *net.UnixListener) {
```

Comment:
Would you mind making the swap to a `linux_windows` and `nolinux_nowindows` pair of buildtagged files here?